### ISSUE TITLE
iviewer ROI Grid view

### ISSUE THREAD

**Issue by will-moore** [openmicroscopy/design#91]
Following on from https://github.com/openmicroscopy/design/issues/88

A "Grid" view of image planes would be useful to show a heat-map of Shapes and allow multi-selection of Planes.
Could use Tabs to select regular Table of ROIs or Grid:

![grid-view1](https://user-images.githubusercontent.com/900055/38303839-1eb180fc-37ff-11e8-8555-83424c2d5a3c.jpg)

With Grid selected, we show a grid of planes, with the current plane highlighted:

![grid-view2](https://user-images.githubusercontent.com/900055/38304676-225775ba-3802-11e8-97ce-9b752e7986dd.jpg)

Selecting a Range by dragging or shift-click selects multiple planes in Grid view.
We enable Z-projection to correspond to Z-range. For T-slider we simply update to the last-clicked T-index in the Grid view.

Users can also use the Z and T sliders (including Z-projection) to select plane(s) and this will sync the Grid view selection.

This will allow copying and pasting of Shapes as described in https://github.com/openmicroscopy/design/issues/90 without needing to wait for T-projection support from the server.
When T-projection is supported, we can also sync T-range with the range selected in the Grid view.

**Comment by will-moore** [comment]
cc @jburel @jrswedlow @waxenegger Could this work in the absence of T-projection?
NB: We should still aim to get T-projection supported ASAP in OMERO.

**Comment by will-moore** [comment]
I don't think this is a new mode, or that it involves any filtering for shapes.
It's really just another way of adjusting the Z & T index/range. So it would have the exact same effect as adjusting the Z & T index/range via the Z/T sliders and Z-projection button.

**Comment by joshmoore** [comment]
Random comment from the peanut gallery: it's nice to see (ever) more information encoded in a quickly comprehensible way.
